src/models/PrimaryAuth.js

```diff
@@ -106,12 +106,21 @@ function (Okta, BaseLoginModel, CookieUtil, Enums) {
 
       var primaryAuthPromise;
 
-      if (this.appState.get('isUnauthenticated') && !this.settings.get('features.passwordlessAuth')) {
-        primaryAuthPromise = this.doTransaction(function (transaction) {
-          var authClient = this.appState.settings.authClient;
-          return this.doPrimaryAuth(authClient, signInArgs, transaction.authenticate);
-        });
-      } else {
+      var authClient = this.appState.settings.authClient;
+
+      if (this.appState.get('isUnauthenticated')) {
+        // bootstrapped with stateToken
+        if (!this.settings.get('features.passwordlessAuth')) {
+          primaryAuthPromise = this.doTransaction(function (transaction) {
+            var authClient = this.appState.settings.authClient;
```

Comment:
and we can get rid of this, since we already have it from line 109 (which should move inside the if isUnathenticated)

Reply:
yes, forgot to remove this